harmony-api/src/repos/followRepo.ts

```diff
@@ -0,0 +1,65 @@
+import prisma from '../../prisma/prisma'
+import { HttpError } from '../models/error/httpError'
+import { Follow } from '../models/follow'
+
+export const followUser = async (followInfo: Follow): Promise<Follow> => {
+  try {
+    const followResult = await prisma.follow.create({
+      data: {
+        followingId: followInfo.followingId,
+        followerId: followInfo.followerId,
+      },
+    })
+
+    return {
+      followingId: followResult.followingId,
+      followerId: followResult.followerId,
+    }
+  } catch (e) {
+    throw new HttpError('Already following user', 400)
+  }
+}
+
+export const unFollowUser = async (followInfo: Follow): Promise<Follow> => {
+  const followResult = await prisma.follow.delete({
+    where: {
+      followingId_followerId: {
+        followingId: followInfo.followingId,
+        followerId: followInfo.followerId,
+      },
+    },
+  })
+  if (followResult === null) {
```

Comment:
Just a heads up, in JS you can just do `if(followResult)`, don't need any changes here though

https://www.freecodecamp.org/news/falsy-values-in-javascript/#:~:text=Description,)%2C%20and%20false%20of%20course.

Reply:
I do this in other places. I dont know why I forgot that it was a thing here 🤔